.github/workflows/docker.yaml

```diff
@@ -0,0 +1,46 @@
+name: Publish Images to ECR
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  Publish:
+    # run only on upstream repo
+    if: github.repository_owner == 'mindsphere'
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+
+    strategy:
+      fail-fast: true
+      matrix:
+        connector: [consumer, provider]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
how do you regenerate these secrets? are they https://docs.github.com/en/actions/security-guides/encrypted-secrets (I cannot verify) ?

Reply:
Yes. They are encrypted secrets at repository level.